src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -1,13 +1,48 @@
 package mate.academy
 
+import mate.academy.HelloWorldAnonymousClasses.Greetings.HELLO_SOMEONE
+import mate.academy.HelloWorldAnonymousClasses.Greetings.HELLO_WORLD
+import mate.academy.HelloWorldAnonymousClasses.Greetings.HOLA_MUNDO
+import mate.academy.HelloWorldAnonymousClasses.Greetings.HOLA_SOMEONE
+import mate.academy.HelloWorldAnonymousClasses.Greetings.SALUT_SOMEONE
+import mate.academy.HelloWorldAnonymousClasses.Greetings.SALUT_TOUT_LE_MONDE
+
 class HelloWorldAnonymousClasses {
+    object Greetings {
```

Comment:
Using 'object' here is overengineering a little. 
If you want to achieve 'static'-behaviour , you may use 'companion object' instead.
However, just const val is OK in this case